python/nano/setup.py

```diff
@@ -79,8 +79,9 @@ def setup_package():
                         "PyTurboJPEG",
                         "opencv-transforms",
                         "onnx",
-                        "onnxruntime"]
-    install_requires = ["intel-openmp"]
+                        "onnxruntime",
```

Comment:
@TheaperDeng we do need onnxruntime here anymore, right?

Reply:
will remove in https://github.com/intel-analytics/BigDL/pull/4497/